system/controllers/topView.py

```diff
@@ -1,11 +1,14 @@
 from django.shortcuts import render_to_response
 from system.models import Equipment, Reserved
+from django.contrib.auth.decorators import *
```

Comment:
__[flake8]__
* [F403] 'from django.contrib.auth.decorators import *' used; unable to detect undefined names